src/shortcuts/bumpThoughtDown.js

```diff
@@ -59,13 +60,15 @@ export default {
           context: unroot(contextOf(context).concat('')),
           rank: rankNew,
           value,
+          uuid: createUuid()
         })
 
         dispatch({
           type: 'setCursor',
           thoughtsRanked: unroot(contextOf(thoughtsRanked).concat({
             value: '',
-            rank
+            rank,
+            uuid: cursor.uuid
```

Comment:
I believe uuid is a property of the child rather than the path as a whole.

Reply:
Thanks for pointing this our. We need to use `headUuuid` here.